packages/volto/src/components/theme/View/AlbumView.jsx

```diff
@@ -18,178 +13,138 @@ import openSVG from '@plone/volto/icons/open.svg';
 import aheadSVG from '@plone/volto/icons/ahead.svg';
 import backSVG from '@plone/volto/icons/back.svg';
 
-/**
- * Album view component class.
- * @function AlbumView
- * @param {Object} content Content object.
- * @returns {string} Markup of the component.
- */
-class AlbumView extends React.Component {
-  constructor(props) {
-    super(props);
-
-    this.state = {
-      openIndex: undefined,
-    };
-
-    this.closeModal = this.closeModal.bind(this);
-    this.nextImage = this.nextImage.bind(this);
-    this.prevImage = this.prevImage.bind(this);
-  }
-
-  closeModal() {
-    this.setState({
-      openIndex: -1,
-    });
-  }
-
-  nextImage() {
-    const openIndex =
-      (this.state.openIndex + 1) % this.props.content.items.length;
-    this.setState({
-      openIndex,
-    });
-  }
-
-  prevImage() {
-    const openIndex =
-      (this.state.openIndex - 1) % this.props.content.items.length;
-    this.setState({
-      openIndex,
-    });
-  }
+const AlbumView = ({ content }) => {
+  const [openIndex, setopenIndex] = useState(undefined);
```

Comment:
Usually in javascript we use camel case, so `setOpenIndex` rather than `setopenIndex`